### DISCUSSION TITLE
Notifications when new translations are required

### DISCUSSION THREAD

**Discussion by lakano** [THM-Health/PILOS#1502] [Ideas]
Hi,
I've just completed some untranslated sentences in french.
I don't known how I could be notified each time there is awaiting translations, do you have any ideas?

**Comment by samuelwei** [comment]
Good idea,I’ll have a look how to do it automatically with poeditor

**Comment by lakano** [comment]
I've already checked in my poeditor website account, I haven't seen any option of this kind. 
But may be as the owner of the project you may have more options than me indeed.
